src/utils/debugging-master/llm-log.php

```
@@ -1,77 +1,127 @@
-		return ($_SERVER['HOME'] ?? $_ENV['HOME'] ?? getenv('HOME') ?: '/tmp')
-			. '/.genesis-tools/debugging-master/sessions';
-	}
+		if (self::$resolvedBase !== null) return self::$resolvedBase;
+		if (self::$probed) return null;
+		self::$probed = true;
 
-	private static function getSessionPath(): string
-	{
-		if (self::$sessionPath === '') {
-			self::$sessionPath = self::sessionsDir() . '/' . self::$currentSession . '.jsonl';
+		$candidates = array_values(array_filter(self::HOSTS, fn($h) => $h !== '' && !str_starts_with($h, '__')));
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $h. Configured minimum length is 3. (undefined)

(ShortVariable)